src/pages/utils.tsx

```diff
@@ -133,3 +133,7 @@ export function isValidAccountAddress(accountAddr: string): boolean {
   // can start with that and see if any fails to parsing
   return /^(0[xX])?[a-fA-F0-9]{1,64}$/.test(accountAddr);
 }
+export function isHex(text: string) {
+  // if it's hex, and is <= (64 + 2 for 0x) char long
+  return text.startsWith("0x") && text.length <= 66;
```

Comment:
This seems similar to the regex on line 134 except this doesn't check that the chars are hex digits. Can you reuse the existing function?

Reply:
I used the same check it used before here https://github.com/aptos-labs/explorer/pull/92/files#diff-971be23a4da4e8d5a19915e6824596394b7b2b2b091130a93a7f3d38d8888a4cL234

Is tnx hex and account address have the same format?